placeholder.json

```diff
@@ -278,6 +278,13 @@
     {
       "Key": "tooltip copied text",
       "Text": "Copied!"
+    },    {
+      "Key": "defaultAuthor",
+      "Text": "Mack Trucks"
+    },
+    {
+      "Key": "defaultReadTime",
```

Comment:
Please ensure these are added to all Mack trucks sharepoint by tomorrow!

Reply:
will do